sdks/common/runner-api/src/main/proto/beam_runner_api.proto

```diff
@@ -525,6 +525,11 @@ message Trigger {
   message Never {
   }
 
+  // Always ready. This can also be expressed as ElementCount(1) but
+  // is more explicit.
+  message Always {
```

Comment:
How about `Whenever` :-)

Yes, I'm quite open to this. I just went dual to `Never`, which anyhow is accurate only insofar as GC firings are not governed by the trigger.